src/Api/InitializeInTracee.cpp

```diff
@@ -4,16 +4,47 @@
 
 #include "Api/InitializeInTracee.h"
 
+#include <functional>
+
 #include "Api/Orbit.h"
-#include "OrbitBase/Result.h"
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/UniqueResource.h"
+#include "UserSpaceInstrumentation/Attach.h"
+#include "UserSpaceInstrumentation/ExecuteInProcess.h"
+#include "UserSpaceInstrumentation/InjectLibraryInTracee.h"
 
+using namespace orbit_user_space_instrumentation;
+using orbit_grpc_protos::ApiTableInfo;
 using orbit_grpc_protos::CaptureOptions;
 
 namespace orbit_api {
+ErrorMessageOr<void> InitializeInTracee(const CaptureOptions& capture_options) {
+  if (capture_options.api_table_infos().size() == 0) {
+    return ErrorMessage("No api table to initilize.");
+  }
+
+  ErrorMessageOr<void> result = outcome::success();
+  int32_t pid = capture_options.pid();
+
+  OUTCOME_TRY(AttachAndStopProcess(pid));
+  {
```

Comment:
The additional scope is not needed here. Restarting on return is just fine.

Reply:
Well that's interesting, if we get rid of the scope, then "result" will always be outcome::success because the scope exit will be executed after the return value has been assigned.

Reply:
Done.